### DISCUSSION TITLE
Support for display p3 color profile.

### DISCUSSION THREAD

**Discussion by ice-cream-coder** [airbnb/lottie-ios#1839] [Ideas]
I saw the announcement about Lottie 4.0 today and it looks awesome! Great work. As far as I can tell, Lottie uses sRGB by default. Has there been any talk about supporting wider gamut color profiles like Display P3?

**Comment by calda** [comment]
There's a feature request for this from a few years ago here: https://github.com/airbnb/lottie-ios/issues/958

Lottie currently uses the device RGB color space ([`CGColorSpaceCreateDeviceRGB()`](https://developer.apple.com/documentation/coregraphics/1408837-cgcolorspacecreatedevicergb)). We could add support for alternative color spaces by adding a configuration point to the `LottieConfiguration` type. This would be a reasonably straightforward change if anyone from the community is interested in working on it.
